packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -210,14 +192,29 @@ describe('importCSV', function () {
       const ns = 'db.col';
 
       const totalRows = analyzeResult.totalRows;
-      const fields = _.mapValues(
-        analyzeResult.fields,
+      const fields = _.mapValues(analyzeResult.fields, (field, name) => {
```

Comment:
Small thing, I like to reduce lodash usage where possible, since we're updating this line can we remove the lodash dependency here? We could do something like 
```const fields = Object.entries(analyzeResult.fields).map(([key, value]) => { ... })```

Reply:
The file already uses _.isPlainObject() :)